Since we ship gtk-doc patches from Debian, we can do this.

--- docs/reference/meson.build
+++ docs/reference/meson.build
@@ -52,7 +52,7 @@ if get_option('gtk_doc')
   endif
   # Check we have the minimum gtk-doc version required. Older versions won't
   # generate correct documentation.
-  dependency('gtk-doc', version : '>=1.32.1',
+  dependency('gtk-doc', version : '>=1.32',
     fallback : ['gtk-doc', 'dummy_dep'],
     default_options : ['tests=false'])
 endif
